modules/sharedaddy/amp-sharing.css

```diff
@@ -0,0 +1,110 @@
+amp-social-share {
+	border-radius: 50%;
+	color: #fff !important;
+    -webkit-border-radius: 50%;
```

Comment:
I'd recommend using tabs for indenting in the whole file, so no one inadvertently changes all spaces into tabs when they next edit this file in their editor.

Reply:
Sorry, this was a bad c/p from the non-AMP stylesheet. Will push fix shortly